cli/pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[build-system]
+requires = ["scikit-build-core>=0.3.3", "pybind11"]
+build-backend = "scikit_build_core.build"
+
+
+[project]
+name = "graphar_cli"
+version = "0.0.1"
+description = "GraphAr command line tool"
+readme = "README.md"
+authors = [{ name = "GraphAr community", email = "dev@graphar.apache.org" }]
+requires-python = ">=3.7"
+dependencies = ["typer ~= 0.12.3", "pydantic ~= 2.8.2", "pyyaml ~= 6.0.2"]
```

Comment:
Why did we pin minor versions? As I remember, all these projects are following semantic-versioning and there shouldn't be a problem if we pin only major version.

Reply:
The `~=` operator will automatically pin the major version number.